deploy-board/run.sh

```diff
@@ -12,9 +17,15 @@ function server_start {
     mkdir -p /tmp/deploy_board
     export ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
     export PYTHONPATH=$ROOT_DIR
-    python $ROOT_DIR/manage.py runserver 0.0.0.0:8888 >& /tmp/deploy_board/access.log &
 
-    echo "Deployboard started. Check log at /tmp/deploy_board/service.log"
+    if [ "$1" == "FOREGROUND" ]
+    then
+        echo "Deployboard started. Check log at /tmp/deploy_board/service.log"
+        python $ROOT_DIR/manage.py runserver 0.0.0.0:8888 >& /tmp/deploy_board/access.log
+    else
+        python $ROOT_DIR/manage.py runserver 0.0.0.0:8888 >& /tmp/deploy_board/access.log &
+        echo "Deployboard started. Check log at /tmp/deploy_board/service.log"
```

Comment:
do you also want to write a pid file somewhere so the daemon watcher can watch it? (like monitd)